packages/utilities/src/hooks/useId.test.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { resetIds } from '../object';
+import { useId } from './useId';
+
+describe('useId', () => {
+  afterEach(() => {
+    resetIds();
+  });
+
+  it('uses the same ID without prefix', () => {
+    const TestComponent: React.FunctionComponent = () => {
+      const id = useId();
+      return <div id={id} />;
+    };
+    const wrapper = mount(<TestComponent />);
+    const firstId = wrapper.getDOMNode().id;
+    // Re-render the component
+    wrapper.update();
```

Comment:
Does update rerender even if there hasn't been prop updates?

Reply:
Yes, at least based on the docs